src/mote/templates/teams.html

```diff
@@ -0,0 +1,314 @@
+                            type="button" onclick="populate_channel_list();">
+                            Meetings by Channel
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" type="button" href="#">
+                            Meetings by Group
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" type="button" href="#">
+                            About
+                        </a>
+                    </li>
+                </ul>
+                <div class="d-flex flex-column justify-content-center align-items-center">
+                    <div class="input-group">
+                        <input type="text" autocomplete="off" class="form-control" placeholder="Search meeting logs"
+                            aria-label="Search meeting logs" aria-describedby="findtext" style="width: 350px;"
+                            id="findtext">
+                    </div>
+                    <div class="input-group" data-bs-auto-close="true">
+                        <ul class="dropdown-menu w-100" id="dropdown">
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <div class="container-lg">
+        <div class="card mt-1" id="bigcard">
+            <div class="card-body pt-0">
+                <div class="mt-4">
+                    <br><br>
+                    <div class="checkbox">
+                        <div class="d-flex justify-content-between">
+                        <div><i class="fa fa-users" aria-hidden="true"></i> Meetings by Group</div>
+                        <div><input class="form-check-input" type="checkbox"> Hide inactive</div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+                    <br><br>
+                    <div class="card">
+                        <div class="card-header">
+                            23 Groups
+                        </div>
+                        <table class="table">
+                            <tbody>
+                                {% for i in teamdata %}
+                                <tr>
+                                    <td class="item">{{i}}</td>
+                                    <td class="item">
+                                        <item>
+                                            Latest Minutes:
+                                        </item>
+                                        <div class="btn-group" role="group"
+                                            aria-label="Basic checkbox toggle button group">
+                                            <input type="checkbox" class="btn-check" id="btncheck1" autocomplete="off">
+                                            <label class="btn btn-outline-primary" for="btncheck1"><i
```

Comment:
All buttons will share the same id, which is not right.
Either make this ID unique, or you could simply remove it since you're not using it anyway.

Reply:
sure